### PR TITLE
Header responsive

### PR DESCRIPTION
**Pull request by EriMiwa**
it's implement of header part for responsive.

@yoniakabecky I wanna consult 2point below.
I try to put header and footer on your page. main contant need to adjust to come next of header.
then, footer part comes widle parts of your page.(Even thought I adjust this part comes bottom)
could you know if these points can be solved?

### CODE REVIEW

File: training/index.html [hunk 1]
```diff
@@ -13,24 +13,49 @@
...
+                      </nav>
+                    </div>
+                  </header>
+                  <head>
+                        <meta charset="UTF-8" lang="en">
+                        <meta name="viewport" content="width=device-width,initial-scale=1">
+                        <title>ballet bloch canada -ballet school vancouver-</title>
+                        <meta name="description" content="Classical ballet and contemporary training for young dancers. The school offers performance and choreographic opportunities with professional artists. Ages 4-18">
+                        <meta name="keywords" content="ballet, istd,classicalballet, danceclasses, balletschool, balletvancouver, balletcanada, balletclasses, tracy-leabloch">
+
```
Comment (yoniakabecky) [inline training/index.html]: why did you add head section inside the body?

File: index.html [hunk 2]
```diff
@@ -2,42 +2,66 @@
 <html>
   <head>
     <meta charset="UTF-8" lang="en">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
     <title>ballet bloch canada -ballet school vancouver-</title>
```
Comment (yoniakabecky) [inline index.html]: can you move title tag below the meta?

**Review by yoniakabecky — CHANGES_REQUESTED** [review]

File: css/style.css [hunk 3]
```diff
@@ -121,11 +137,144 @@ footer {
...
+  padding-top: 30px;
+  padding-bottom: 30px; 
+}
+
+.f-social-area img {
+  margin-bottom: 20px;
+}
+
+}
+
```
Comment (yoniakabecky) [inline css/style.css]: extra end bracket...

File: index.html (hunk 2 above)
Comment (EriMiwa) [reply index.html]: OK!

File: training/index.html (hunk 1 above)
Comment (EriMiwa) [reply training/index.html]: Sorry! it's my fault! I changed it!

File: css/style.css (hunk 3 above)
Comment (EriMiwa) [reply css/style.css]: it's a end bracket for media query. but my indents were terrible. so adjusted it ;)
thanks!

File: css/training.css [hunk 4]
```diff
@@ -8,6 +8,10 @@ body {
     color: #FEFBFB;
 }
 
+.main-area {
+    margin-top: 95px;
```
Comment (EriMiwa) [inline css/training.css:12]: it's for avoiding to hide top image under the menu bar.

File: css/training.css [hunk 5]
```diff
@@ -105,7 +105,20 @@ h1:after {
     background-color: #73545A;
     color: #F6EDE4;
 }
-
+.main {
+    min-height: 100vh;
+    position: relative;/*←相対位置 */
+  }
+  footer {
+    position:absolute;
```
Comment (yoniakabecky) [inline css/training.css]: What is this footer section? Is there any difference between main style.css and this training.css?

File: css/training.css [hunk 6]
```diff
@@ -105,7 +105,20 @@ h1:after {
     background-color: #73545A;
     color: #F6EDE4;
 }
-
+.main {
+    min-height: 100vh;
+    position: relative;/*←相対位置 */
+  }
```
Comment (yoniakabecky) [inline css/training.css]: I think this is unnecessary...

**Review by yoniakabecky — CHANGES_REQUESTED** [review]

**Review by yoniakabecky — APPROVED** [review]
